docs/content/documentation/schema/_index.md

```diff
@@ -33,43 +33,76 @@ Each OSCAL model is represented in multiple, machine readable *formats* (e.g., X
 
 ## The OSCAL Layers
 
-The following layers are defined in the OSCAL architecture, which are listed from the bottom up.
+The diagram above identifies the OSCAL layers from the bottom up because each higher layer relies on the layer beneath it. The following OSCAL layer discussion begins with the bottom layer in the diagram and address each in order moving toward the top of the diagram: 
+- Catalog layer
+- Profile layer
+- Implementation Layer
+- Assessment Layer
+- Assessment Results Layer
 
 ### Catalog Layer
```

Comment:
Maybe change these headings to follow the pattern "Catalog Layer Overview"? That way there is a differentiation between the lower layer pages in the side nav.